src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/StringToListConverter.shared.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using Xamarin.CommunityToolkit.Extensions.Internals;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	/// <summary>
+	/// Returns a string array that contains the substrings in this string that are delimited by <see cref="Separator"/>.
+	/// </summary>
+	public class StringToListConverter : ValueConverterExtension, IValueConverter
+	{
+		/// <summary>
+		/// A string that delimits the substrings in this string.
+		/// </summary>
+		public string Separator { get; set; } = string.Empty;
```

Comment:
Would be great if we could specify several separators as well, but not sure how to implement it better...

Reply:
@AndreiMisiukevich when you have a chance, could you please have a look at my implementation? I have implemented Separator (single string) and Separators (multiple strings), they are going to be applied simultaneously if exist.
```
<xct:StringToListConverter
    x:Key="StringToListConverter"
    Separator=","
    SplitOptions="RemoveEmptyEntries">
    <xct:StringToListConverter.Separators>
        <x:String>.</x:String>
        <x:String>;</x:String>
    </xct:StringToListConverter.Separators>
</xct:StringToListConverter>
```